content/ja/handbook/engineering/releases/monthly-releases/index.md

```
@@ -0,0 +1,253 @@
+
+マンスリーリリースプロセスに関わる手順の概要：
+
+![マンスリーリリース概要](/images/engineering/releases/monthly-releases/monthly-release-overview.jpg)
+
+* [ダイアグラムソース](https://docs.google.com/presentation/d/1YRjA1dYCXNXp06VltDYlik1MdFyzUvaeXKk69mMPcA4/edit#slide=id.g2951f7d5d31_1_0)
+
+エンドツーエンドのプロセスは以下の段階で構成されています。
+
+1. **最初のステップ** - マンスリーリリースのセットアップ、リリーススケジュールとデプロイケイデンスの設定を含む初期ステップ。
+1. **GitLab.comへのデプロイ** - マイルストーンの開始からリリース日の1週間前まで、GitLab.comは1日に複数回デプロイを受け取ります。アプリケーションの変更がマンスリーリリースに含まれるためには、GitLab.comに正常にデプロイされる必要があります。
+1. **リリース候補** - テスト用[リリース候補（RC）](#what-is-a-release-candidate-and-when-are-they-created)が作成され、対象とする[semver](https://semver.org)バージョンのステーブルブランチとともに作成されます。リリース候補パッケージがビルドされ、テストされ、[pre環境](/handbook/engineering/infrastructure-platforms/environments/#pre)にデプロイされます。成功した結果は、このパッケージが最終バージョンとして使用できることを示します。この時点で[リリースマネージャー](https://about.gitlab.com/community/release-managers/)がリリースに含まれる最終コミットをアナウンスします。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)